tests/10-rdfc-create.js

```diff
@@ -30,188 +30,189 @@ const {match} = endpoints.filterByTag({
 const verifier = localVerifier({cryptosuite});
 
 describe('ecdsa-rdfc-2019 (create)', function() {
-  describe('ecdsa-rdfc-2019 (issuers)', function() {
-    this.matrix = true;
-    this.report = true;
-    this.implemented = [];
-    this.rowLabel = 'Test Name';
-    this.columnLabel = 'Implementation';
-    for(const [name, {endpoints: issuers}] of match) {
-      // test for each support key type
-      for(const keyType of keyTypes) {
-      // loop through each issuer in suite
-        for(const issuer of issuers) {
-          const {supportedEcdsaKeyTypes} = issuer.settings;
-          // if an issuer does not support the current keyType skip it
-          if(!supportedEcdsaKeyTypes.includes(keyType)) {
-            continue;
-          }
-          // add implementer name and keyType to test report
-          this.implemented.push(`${name}: ${keyType}`);
-          describe(`${name}: ${keyType}`, function() {
-            let issuedVc;
-            let proofs;
-            const verificationMethodDocuments = [];
-            before(async function() {
-              issuedVc = await createInitialVc({
-                issuer,
-                vc: credentials.create.document
+  for(const vcVersion of vectors.vcTypes) {
+    describe(`ecdsa-rdfc-2019 (issuers) VC ${vcVersion}`, function() {
+      this.matrix = true;
+      this.report = true;
+      this.implemented = [];
+      this.rowLabel = 'Test Name';
+      this.columnLabel = 'Implementation';
+      for(const [name, {endpoints: issuers}] of match) {
+        // test for each support key type
+        for(const keyType of vectors.keyTypes) {
+        // loop through each issuer in suite
+          for(const issuer of issuers) {
+            // does the endpoint support this test?
+            if(!endpointCheck({endpoint: issuer, keyType, vcVersion})) {
```

Comment:
I'll take it as a complement that you left my endpointCheck function in there. Do we still throw if an implementation doesn't support a keyType in another test? We probably want to treat keyType support the same way in all tests. That can be an issue addressed later as this PR is beyond huge.

Reply:
Yeah, this is in the issuer side of the tests, which hasnt been ported yet to the helper, mainly because there are a few quirks in how issuer tests handle test vector data that we need to work through (hard to tell on first glance which variances to verifier tests are accidental draft vs actual differences).